registry/sql-registry/registry/db_registry.py

```diff
@@ -405,16 +405,18 @@ def _fill_entity(self, e: Entity) -> Entity:
         return e
 
     def _get_edges(self, ids: list[UUID], types: list[RelationshipType] = []) -> list[Edge]:
-        if not ids or not types:
+        if not ids:
```

Comment:
Shouldn't we just return an empty list if no input `types` is specified?

Reply:
Based on the line 94 where this method is called "all_edges = self._get_edges(ids)", and the sql query on line 410 when the types is empty, my understanding is when types is empty we should query for results containing all relationship types. Otherwise, the 'all_edges' will return empty list which is the root cause of this error. We can also set all relationship types at line 94, but I think it will be less efficient.